examples/lines_bars_and_markers/scatter_star_poly.py

```diff
@@ -1,40 +1,48 @@
 """
-=================
-Scatter Star Poly
-=================
+===============
+Marker examples
```

Comment:
I'm not a big fan of this title: it doesn't summarizes what the example showcases in particular compared to all of the other examples on markers.

Reply:
I agree that it's not very instructive. But I didn't find a better name. OTOH, the example itself doesn't add much compared to the maker reference and module docs. I was about to delete it but decided to keep it as a usage example.

If you can think of a better name, proposals are always welcome.

Reply:
I'd be in favor of removing it :)